src/components/Type/Type.js

```diff
@@ -115,6 +115,7 @@ Type.COLORS = {
   // Grayscale
   GRAY_PRIMARY: COLORS.GRAY_PRIMARY,
   GRAY_SECONDARY: COLORS.GRAY_SECONDARY,
+  GRAY_STROKE_AND_DISABLED: COLORS.GRAY_STROKE_AND_DISABLED,
```

Comment:
Yeah, so this is fine for now and adding needed classes is also certainly fine. But  just a heads up we're going to make all this use CSS Modules soon, so all this will hopefully get rolled up into that soon.